src/app/service/in-memory-document.service.ts

```diff
@@ -0,0 +1,81 @@
+import { Injectable } from '@angular/core';
+import { Observable, of, BehaviorSubject } from 'rxjs';
+import { Collaborator } from '../model/collaborator';
+import { DocumentService, DocumentModel } from './document.service';
+import { Card, Point, Arrow } from '../model';
+
+// TODO
+export class InMemoryDocumentModel {
+  cards: Card[] = [];
+  arrows: Arrow[] = [];
+
+  constructor(public id: string) {}
+  addCard(card: Card) { this.cards.push(card); }
+  addArrow(arrow: Arrow) { this.arrows.push(arrow); }
+}
+
+@Injectable()
+export class InMemoryDocumentService extends DocumentService {
```

Comment:
(not contesting at all, just curious)

So gameplan is to start out with uxgraphs all be in-memory, no collaboration?